### DISCUSSION TITLE
Welcome to Ballerine Discussions!

### DISCUSSION THREAD

**Discussion by alonp99** [ballerine-io/ballerine#16] [Announcements]
## 👋 Welcome!
  Discussions ןs a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zhangpengchen** [comment]
Hello, first of all thanks for open source a great project, the functionalities and technical setup looks very interesting, just wondering if it's partially open source or contains all features. For example under packages and services folders are linking to some private repositories such as ballerina-backend.

**Reply by alonp99** [reply]
Hey @zhangpengchen , the nested sub-modules are either in development or in the process of migration to the open-source monorepo, we don’t want to open-source half-baked code just yet.
We are working hard to complete a suite of products that answer a wide variety of use-cases. ill be glad to hear about your use case maybe we can already help with it. (you can find us on the community slack/discord :))

**Reply by zhangpengchen** [reply]
Thanks @alonp99 I will follow up the repository, once the git modules are in place, if there are use cases you will hear from me.

**Comment by waqas-ashraf-venturedive** [comment]
There are services using the ```UNIFIED_API_URL``` but not able to find the implementation in this repo. That is what you mentioned above; like it's WIP and you will open source it or that is something private?

Thanks
